test/index.test.js

```diff
@@ -95,20 +101,11 @@ describe('config parser', () => {
                 assert.deepEqual(data.jobs.main[0].secrets, []);
                 assert.deepEqual(data.jobs.main[0].environment, {});
                 assert.strictEqual(data.jobs.main[0].commands[0].name, 'config-parse-error');
-                assert.match(data.jobs.main[0].commands[0].command, /ValidationError:.*"jobs" is required/);
-                assert.match(data.errors[0], /ValidationError:.*"jobs" is required/);
+                assert.match(data.errors[0], 'ValidationError: "jobs" is required');
             }));
     });
 
     describe('structure validation', () => {
-        describe('overall config', () => {
-            it('returns an error if missing jobs', () =>
```

Comment:
I think we need to keep this test and modify it to say `returns an error if missing jobs and no template specified`

Reply:
yeah should add it back, i deleted it when i placed `merge` after `structure validation`. now `merge` happens before `structure validation` so it should pass now